_inc/client/security/manage-plugins.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import Card from 'components/card';
+import { withModuleSettingsFormHelpers } from 'components/module-settings/with-module-settings-form-helpers';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+
+export const ManagePlugins = withModuleSettingsFormHelpers(
+	class extends Component {
+		trackOpenCard = () => {
+			analytics.tracks.recordJetpackClick( {
+				target: 'foldable-settings-open',
+				feature: 'manage-plugins',
+			} );
+		};
+
+		trackClickConfigure() {
+			analytics.tracks.recordJetpackClick( {
+				target: 'configure-plugins',
+				page: 'plugins-manage',
+			} );
+		}
+
+		render() {
+			const configLink = () => {
```

Comment:
Can we make `configLink` be a class method instead ?

Reply:
Done in e2bd16022cabef9f9f0c111c9c62093d4e3b44bc